### PR TITLE
Cerberus address moved to genesis

### PR DESCRIPTION
**Pull request by megaherz**

### CODE REVIEW

File: x/pdv/handler.go [hunk 1]
```diff
@@ -9,11 +9,16 @@ import (
...
+	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
 // NewHandler creates an sdk.Handler for all the pdv type messages
-func NewHandler(cerberus cerberusapi.Cerberus, keeper Keeper) sdk.Handler {
+func NewHandler(keeper Keeper) sdk.Handler {
+	pk := secp256k1.PrivKeySecp256k1{}
+
 	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
+		cerberus := cerberusapi.NewClient(keeper.GetCerberusAddr(ctx), pk)
```
Comment (megaherz) [inline x/pdv/handler.go:20]: @mikluke need to check performance

File: app/app.go [hunk 2]
```diff
@@ -304,9 +304,9 @@ func NewDecentrApp(
 		bank.ModuleName,
 		slashing.ModuleName,
 		pdv.ModuleName,
-		token.ModuleName,
 		profile.ModuleName,
 		community.ModuleName,
+		token.ModuleName,
```
Comment (megaherz) [inline app/app.go:309]: @mikluke order changed

File: x/pdv/handler.go (hunk 1 above)
Comment (mikluke) [reply x/pdv/handler.go:20]: lets create client in NewHandler

Comment (mikluke) [reply x/pdv/handler.go:20]: so also we need some mechanism to react on keeper.SetCerberus

maybe we should create a client in the keeper?

File: x/pdv/handler.go [hunk 3]
```diff
@@ -25,6 +28,21 @@ func NewHandler(cerberus cerberusapi.Cerberus, keeper Keeper) sdk.Handler {
 	}
 }
 
+var (
+	prevCerberusAddress string
+	cerberus            cerberusapi.Cerberus
+)
+
+func GetCerberus(ctx sdk.Context, keeper Keeper) cerberusapi.Cerberus {
```
Comment (megaherz) [inline x/pdv/handler.go:36]: @mikluke, should work, the message loop is likely one threaded.

**Pull request merged**